### PR TITLE
Use frame width for plot scaling

### PR DESCRIPTION
**Pull request by kmshin1397**
Closes fritz-marshal/fritz-beta-feedback#91
Should also close #1803 
plot

Use the `frame_width` parameter instead of `width` in the photometry/spectroscopy plots to remove the legend from width/aspect ratio calculations. Also some other minor tweaks to the plot layouts (spectroscopy sliders are slightly narrower to account for padding; labels wrap to next line a little earlier)

### CODE REVIEW

File: skyportal/plot.py [hunk 1]
```diff
@@ -342,7 +342,7 @@ def photometry_plot(obj_id, user, width=600, device="browser"):
 
     labels = []
     for i, datarow in data.iterrows():
-        label = f'{datarow["instrument"]}/{datarow["filter"]}'
+        label = f'{datarow["instrument"]}/{datarow["filter"]}asfasdfasfsafdsa'
```
Comment (acrellin) [inline skyportal/plot.py:345]: Is this supposed to be here?

**Review by acrellin — APPROVED** [review]
Aside from the possible debug code noted above, :+1:

File: skyportal/plot.py (hunk 1 above)
Comment (kmshin1397) [reply skyportal/plot.py]: Oops, no that's leftover test code good catch

**Review by scizen9 — APPROVED** [review]
Thanks @kmshin1397!  Lot of work to solve this problem.

**Pull request merged**